homeassistant/components/recorder/models.py

```diff
@@ -198,6 +210,36 @@ def to_native(self, validate_entity_id=True):
             return None
 
 
+class Statistics(Base):  # type: ignore
+    """Statistics."""
+
+    __table_args__ = {
+        "mysql_default_charset": "utf8mb4",
+        "mysql_collate": "utf8mb4_unicode_ci",
+    }
+    __tablename__ = TABLE_STATISTICS
+    id = Column(Integer, primary_key=True)
+    created = Column(DATETIME_TYPE, default=dt_util.utcnow)
+    source = Column(String(255), index=True)
```

Comment:
128 chars is enough too. Maybe even 64 or 32 ?

Reply:
I don't think that we care about an index on this btw.

Reply:
Fixed in 4fb7726